### ISSUE TITLE
How can we use VictoryLegend?

### ISSUE THREAD

**Issue by sahil290791** [FormidableLabs/victory-native#85]
I am unable to find any examples on usage of `<VictoryLegend />`?

I have tried this, 
```
<VictoryLegend
     data={[{ name: 'Series 1', symbol: { type: 'star' } }, { name: 'Series 2', labels: { fill: 'blue' } }]}
/>
```
but this doesn't work.

**Comment by JCMais** [comment]
VictoryLegend was not implemented yet on the native version.

**Comment by chrisbolin** [comment]
@JCMais you are totally correct. Sorry for the bad docs @sahil290791. I'll fix that!

**Comment by chrisbolin** [comment]
@sahil290791 VictoryLegend is now available as of version `victory-native@0.12.0`, thanks to @boygirl 💯

**Comment by OzoTek** [comment]
@chrisbolin Hello, is there an issue with `VictoryLegend` with Android ? It works well on iOS but nothing is displayed on Android 
iOS :
![simulator screen shot 1 aug 2017 15 34 34](https://user-images.githubusercontent.com/6436053/28830566-0900e4ec-76cf-11e7-9d17-b45c8c8c3ed9.png)

Android :
![screenshot_20170801-153425](https://user-images.githubusercontent.com/6436053/28830588-19cdee82-76cf-11e7-8097-25711b99c003.png)

```javascript
<VictoryLegend
    data={
        data
            .map((d) => ({
                count: d.y,
                name: d.x,
                symbol: {fill: d.fill},
            }))
            .sort((a, b) => b.count - a.count)
    }
/>
```
